SeleniumAutomationGenerator/ComponentFactorySteps/IComponentFactoryCreatorStep.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+using Core;
+using Core.Models;
+
+namespace SeleniumAutomationGenerator.ComponentFactorySteps
+{
+    public interface IComponentFactoryCreatorStep : IComponentFactoryStep
+    {
+        IEnumerable<ComponentGeneratorOutput> InvokeStep(AutoElementData rootElement, IComponentFileCreator parent, IComponentFactoryCreatorStep next = null);
```

Comment:
lets do that refactoring after I can better understand the tests and we have better code coverage...